construction-event-handler.lua

```diff
@@ -30,12 +31,17 @@ local function on_built(event)
 
         invalidateSpecialBuildingsList(city, entity.name)
 
+        -- TODO: this could be dropped in favor of Util.getGlobalBuilding()
         if global.tycoon_entity_meta_info == nil then
             global.tycoon_entity_meta_info = {}
         end
         global.tycoon_entity_meta_info[entity.unit_number] = {
             cityId = city.id
         }
```

Comment:
`global.tycoon_entity_meta_info` is now being a (subset-) duplicate of `global.tycoon_city_buildings`, but i've just marked it with TODO to keep this PR as small as possible to be tested and merged asap